docs/xtrabackup-version-numbers.md

```diff
@@ -0,0 +1,34 @@
+# Understand version numbers
+
+A version number identifies the product release and any updates. The product contains the latest Generally Available (GA) features at the time of that release. If a feature must be modified after the release, the modification can be included in a new release.
+
+<style>
+    table {
+        border-collapse: collapse;
+        width=100%;
+    }
+    table td {
+        border: 2px solid black;
+        padding: 8px;
+        text-align: center;
+    }
+    tr:nth-child(even){
+        background-color:#f5f5f5
+    }
+</style>
+
+| 8.0.30 | -23 |
+|---|---|
+| Base version | Minor |
+
+Percona uses semantic version numbering, which follows the pattern of base version and minor version. Percona assigns unique, non-negative integers in increasing order for each version release.
+
+The version numbers for Percona XtraBackup 8.0.30-23 define the following information:
+
+* Base version - the leftmost number indicates the version of the Percona XtraBackup that is based on [MySQL 8.0](https://dev.mysql.com/doc/relnotes/mysql/8.0/en/). A base version increase resets the minor version to 0. 
+
+* Minor version - an internal number that denotes the version of the software. A minor version increases by one each time the Percona XtraBackup is released.
+
+Percona XtraBackup 8.0.28-20 and 8.0.28-21 are multiple releases based on MySQL 8.0.28.
+
+
```

Comment:
Add a reference to the blog
https://www.percona.com/blog/2020/08/18/aligning-percona-xtrabackup-versions-with-percona-server-for-mysql/
and mention that the version numbering changed after 8.0.14

Reply:
Updated